### ISSUE TITLE
Tiling becomes empty after reduction

### ISSUE THREAD

**Issue by ulfarsson** [PermutaTriangle/Tilings#439]
```py
import requests
from comb_spec_searcher import CombinatorialSpecification

uri = "https://api.permpal.com/garpur_run/623f1a83a256a078776738af"
data = requests.get(uri).json()
spec = CombinatorialSpecification.from_dict(data["specification"])
spec.show(verbose=True)
```

Look at the reduction from tiling 13.

**Comment by jaypantone** [comment]
<img width="451" alt="Screen_Shot_2022-03-26_at_14 24 45" src="https://user-images.githubusercontent.com/2925902/160244120-f82d870e-8c79-4903-a252-be1a7d6b9e35.png">
The cell that gets reduced to monotone increasing also has a 321 local requirement, hence the whole tiling is empty. I think we should instead be moving local requirements to the other child.

**Comment by jaypantone** [comment]
This is probably fixed by #437
